test/e2e/integration/navigation-test/navigation.ts

```diff
@@ -116,21 +116,21 @@ describe('navigation', () => {
       cy.get(appFrontend.errorReport).should('not.exist');
 
       isUsingDialog && cy.showNavGroups();
-      cy.navGroup('Utfylling', 'Biler').find(ICON_COMPLETE).should('not.exist');
-      cy.navGroup('Utfylling', 'Biler').find(ICON_ERROR).should('not.exist');
-      cy.gotoNavGroup('Utfylling', 'Biler');
+      cy.navGroup('Utfylling', 'Kjøretøy').find(ICON_COMPLETE).should('not.exist');
+      cy.navGroup('Utfylling', 'Kjøretøy').find(ICON_ERROR).should('not.exist');
+      cy.gotoNavGroup('Utfylling', 'Kjøretøy');
 
       isUsingDialog && cy.showNavGroups();
       cy.navGroup('Utfylling', 'Alder').find(ICON_COMPLETE).should('be.visible');
       cy.navGroup('Utfylling', 'Alder').find(ICON_ERROR).should('not.exist');
       cy.gotoNavGroup('Utfylling', 'Fødselsdag');
 
       isUsingDialog && cy.showNavGroups();
-      cy.navGroup('Utfylling', 'Biler').find(ICON_COMPLETE).should('not.exist');
-      cy.navGroup('Utfylling', 'Biler').find(ICON_ERROR).should('not.exist');
+      cy.navGroup('Utfylling', 'Kjøretøy').find(ICON_COMPLETE).should('not.exist');
+      cy.navGroup('Utfylling', 'Kjøretøy').find(ICON_ERROR).should('not.exist');
       isUsingDialog && cy.hideNavGroups();
 
-      cy.findByRole('textbox', { name: /Fødselsdag/ }).type('01011983');
+      cy.findByRole('textbox', { name: /Fødselsdag/ }).type('09061934');
```

Comment:
Nice :sunglasses: (I had to look that up! :sweat_smile:)

Reply:
I had to lie about Scrooge McDuck's birthday unfortunately, since I had set up a minimum date of `01.01.1900` 😆